source/Entity.ts

```diff
@@ -23,7 +23,7 @@ export class Entity implements MonsterData, Partial<GMonster> {
 
     public target: string
 
-    public abilities: { [T in SkillName]?: any } = {}
+    public abilities: { [T in SkillName]?: never } = {}
```

Comment:
I choose never here, since you don't care about any of the typings in this file.